build.gradle

```diff
@@ -375,7 +394,7 @@ subprojects {
     exclude('**/*.xml')
   }
 
-  def publishRegularArtifacts = projectsToPublish.containsKey(project.path)
+  def publishRegularArtifacts = projectsToPublish.contains(project.path)
```

Comment:
Better to use type-agnostic expression (i.e `path in projectsToPublish`).

Reply:
Done